tests/languages/diff/diff_feature.test

```diff
@@ -9,13 +11,14 @@
 ----------------------------------------------------
 
 [
-	["diff", "!     qt: core"],
-	["deleted", "-    qt: core"],
-	["inserted", "+    qt: core gui"],
-	["deleted", "< qt: core"],
-	["inserted", "> qt: core quick"]
+	["diff", "!     qt: core\r\n"],
+	["unchanged", "  unchanged\r\n"],
+	["deleted-sign", "-    qt: core\r\n"],
+	["inserted-sign", "+    qt: core gui\r\n"],
+	["deleted-arrow", "< qt: core\r\n"],
+	["inserted-arrow", "> qt: core quick"]
 ]
 
 ----------------------------------------------------
 
-Checks for deleted, inserted and different lines.
\ No newline at end of file
+Checks for deleted, inserted and different lines.

```

Comment:
Can we get tests for inside diff highlighting?

Reply:
What do you mean with "inside diff highlighting"?
Do you mean a test where we actually highlight the code?

Reply:
Yup!

Reply:
I believe that means add&nbsp;a&nbsp;test like&nbsp;this, except&nbsp;for&nbsp;**Markdown**:
https://github.com/PrismJS/prism/blob/master/tests/languages/markup!%2Bjavascript/javascript_inclusion.test

Reply:
Can I even [write a test](https://prismjs.com/test-suite.html#writing-specific-tests)?

I need to the test runner to load `diff` but use `diff-xxxx` to highlight some given test code. I don't think it's capable of doing that right now, is it?

Reply:
You could even say that this is less a problem of the test runner and more a problem of my PR. I basically create language definitions on the fly. No part of Prism is expecting this.

Reply:
Is it worth extracting to another language, like `diff-extras` or something? Maybe worth considering what our philosophy should be for what's included in the base language vs an extension.

Reply:
I was thinking of maybe making this a plugin, so I would also be able to include the custom CSS I used for the [third screenshot](https://github.com/PrismJS/prism/pull/1889#issue-278797097).

Reply:
Yeah, I like that idea. That addition would be really nice.

Reply:
Alright, a plugin it is then!

Reply:
Regarding the testing: I'll just add it to the list in #1970 when it's merged.